src/components/templates/Audience/Audience.tsx

```diff
@@ -365,27 +414,34 @@ export const Audience: React.FunctionComponent = () => {
           className="audience-container"
           style={{ backgroundImage: `url(${venue.mapBackgroundImageUrl})` }}
         >
-          <div className="video-container">
-            <div className="video">
-              <iframe
-                className={videoFrameClasses}
-                src={iframeUrl}
-                title="Video"
-                frameBorder="0"
-                allow={IFRAME_ALLOW}
-                allowFullScreen
-              />
-            </div>
-            {venue.showReactions && (
+          <div className="audience">
+            <div className="audience-overlay">
               <div
-                className={`reaction-container ${userSeated ? "seated" : ""}`}
+                ref={focusElementOnLoad}
+                className="video-container"
+                style={videoContainerStyles}
               >
-                {userSeated ? renderReactionsContainer() : renderInstructions()}
+                <div className="video">
+                  <iframe
+                    className={videoFrameClasses}
+                    src={iframeUrl}
+                    title="Video"
+                    frameBorder="0"
+                    allow={IFRAME_ALLOW}
+                    allowFullScreen
+                  />
+                </div>
+
+                {venue.showReactions && (
+                  <div className={reactionContainerClassnames}>
+                    {userSeated
+                      ? renderReactionsContainer()
+                      : renderInstructions()}
+                  </div>
+                )}
```

Comment:
It might be worth checking how this looks when `venue.showReactions` is turned off too..

Reply:
Looks good
![image](https://user-images.githubusercontent.com/63194656/110109865-b179c700-7db6-11eb-8e64-a84abde0f57a.png)

Reply:
Yay!